packages/abi-coder/test/utils/constants.d.ts

```diff
@@ -0,0 +1,38 @@
+export declare const U8_MAX: number;
+export declare const U8_MAX_ENCODED: Uint8Array;
```

Comment:
@maschad Just out of curiosity, why do we need these constants?

Reply:
I think this was added from a very old commit, they aren't needed.